lib/laa_crime_schemas/structs/applicant.rb

```diff
@@ -4,6 +4,7 @@ module LaaCrimeSchemas
   module Structs
     class Applicant < Person
       attribute :date_of_birth, Types::JSON::Date
+      attribute :benefit_type, Types::BenefitType
```

Comment:
We need to use `attribute?` otherwise old applications without this attribute will not conform to the Struct.

Reply:
Also, if we are going to send the attribute always, but there is a possibility it can be `nil` (depending on journey taken by provider) we would need `Types::BenefitType.optional`

Reply:
Thank you! Both are the case